packages/locales/README.md

```diff
@@ -0,0 +1,119 @@
+# Translations
+
+Translations are an important part of our design system, allowing us to provide a consistent user experience across different languages and regions. 
+In this document, we will outline the process for adding translations to our components, including the tools and libraries we use, as well as the structure of our translation files.
+
+For the translations in Sanoma Learning Design System, we use [Lit Localize](https://lit.dev/docs/localization/overview/) library.
+
+Translations are provided in the `locales` folder. The translations are stored in `xlif` files, one for each language.
+The file names are the language codes (e.g., `en.json`, `nl.json`, etc.). 
+Each `xlif` file contains key-value pairs, where the keys are the same across all languages, and the values are the translated strings.
+
+## What do example translation look like?
+
+For the translations we use our own message ids, not those generated automatically by Lit Localize.
+This helps to prevent overriding translations accidentally.
+For more details, see: [Lit Localization documentation](https://lit.dev/docs/localization/overview/#overriding-ids).
+
+### Example of a translation
+
+In the component:
+
+```html
+msg('Close', {id: 'sl.inlineMessage.closeButtonLabel’});
+```
+
+In the xliff file:
+
+```xml
+<trans-unit id="sl.inlineMessage.closeButtonLabel">
```

Comment:
This doesn't use the latest conventions?

Reply:
good catch :)